docs/modules/data-structures/pages/vector-search-overview.adoc

```diff
@@ -231,3 +231,8 @@ If using partitions that are larger than the recommended size, ensure that you h
 To decrease pressure on heap memory, you can decrease the number of parallel migrations using `hazelcast.partition.max.parallel.migrations` and `hazelcast.partition.max.parallel.replications`.
 ====
 
+== Tuning tips
+
+1. In Hazelcast 5.5 for searches with small `topK` (e.g. 10) it may be beneficial to artificially increase `topK`, adjust `partitionLimit` accordingly and discard extra results. For initial `topK=10` a good starting point could be `topK` increased to 100 and `partitionLimit` between 50 and 100. While this will make the search slower, it will also improve quality, sometimes significantly. Overall this setup can be more efficient than increasing index build parameters (`max-degree`, `ef-construction`) which results in slower index build and searches. With very small `topK` or `paritionLimit` the search algorithm is less able to escape local minima and find best results.
```

Comment:
```suggestion
1. In Hazelcast 5.5 for searches with small `topK` (for example, 10) it may be beneficial to artificially increase `topK`, adjust `partitionLimit` accordingly, and discard extra results. While `topK=10` is a good starting point, you could increase `topK` to 100 and set `partitionLimit` to between 50 and 100. While this will make the search slower, it will also improve quality, sometimes significantly. Overall, this setup can be more efficient than increasing index build parameters (`max-degree`, `ef-construction`) which results in slower index build and searches. With a very small `topK` or `paritionLimit`, the search algorithm is less able to escape local minima and find the best results.
```

Reply:
```suggestion
1. In Hazelcast 5.5 for searches with small `topK` (for example, 10) it may be beneficial to artificially increase `topK`, adjust `partitionLimit` accordingly, and discard extra results. If you need 10 results a good starting point for tuning could be `topK=100` and `partitionLimit` between 50 and 100. While this will make the search slower, it will also improve quality, sometimes significantly. Overall, this setup can be more efficient than increasing index build parameters (`max-degree`, `ef-construction`) which results in slower index build and searches. With a very small `topK` or `paritionLimit`, the search algorithm is less able to escape local minima and find the best results.
```

my updated suggestion. The idea is that if you need 10 results, request 100 and discard 90 and use that setup as starting point for further tuning. This is counterintuitive and we should improve that in the next version

Reply:
Fantastic @k-jamroz, yes, go with your updated suggestion. Thank you.